website/docs/r/log_parsing_rule.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_log_parsing_rule"
+sidebar_current: "docs-newrelic-resource-log-parsing-rule"
+description: |-
+Create and manage Log Parsing Rule.
+---
+
+# Resource: newrelic\_log\_parsing\_rule
+
+Use this resource to create, update and delete New Relic Log Parsing Rule.
+
+## Example Usage
+
+Use this example to create the log parse rule.
+```hcl
+
+resource "newrelic_log_parsing_rule" "foo"{
+    account_id  = 12345
```

Comment:
No magic numbers, please remote the account_id. It should be optional anyway.

Reply:
removed account Id